### DISCUSSION TITLE
Submitted Korean translation (at weblate)

### DISCUSSION THREAD

**Discussion by jomunim** [UnicornsOnLSD/finamp#898] [Translations]
Hello. First of all, I'm so happy to have discovered the Finamp app.
I realized that this great app doesn't have Korean language yet, so I submitted it.
I'm new to the Github and Weblate system, so there may be some clumsiness. But I've put enough thought into the translation to register.

I'm using the Stable version on my iPhone and the Redesign Beta version on my Android sub phone.
I'm looking forward to the official release of the Redesign version soon.

**Comment by Chaphasilor** [comment]
Hey @jomunim, we haven't integrated a translations update in a while, but I talked about this to @jmshrv just yesterday, and we're planning to integrate the new translations into the stable and beta versions soon!  
We'll also try to make it possible to translate the newly added labels in the beta, those haven't been available through weblate yet.

**Reply by jomunim** [reply]
Ok. I would like to continue to help with Koreanization in a small way.

**Reply by Chaphasilor** [reply]
Your updates translations should be in the stable version since the latest stable update.  
We haven't gotten around to integrating them into the beta yet, sorry!
